### ISSUE TITLE
Pulmonary (?) oligemia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6699]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Oligemia is a reduction in pulmonary blood volume. Most frequently, this reduction is regional, but occasionally it is generalized. Regional oligemia is usually associated with reduced blood flow in the oligemic area. Oligemia appears as a regional or widespread decrease in the size and number of identifiable pulmonary vessels, which is indicative of less than normal blood flow.

PMID:18195376

Comment
See Figure 44 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
